workers/tasks/export/styles/printDocument.scss

```diff
@@ -1,19 +1,37 @@
-/* Editor Base Without Figures As Of 12/2019 */
-
-@import './editorBase.scss';
+/* Imports the equivalent of components/Editor/styles/base.scss,
+but without the media query mixin, which breaks PagedJS-cli:
+https://gitlab.pagedmedia.org/tools/pagedjs-cli/issues/11 */
+@import '~prosemirror-view/style/prosemirror';
+@import '~prosemirror-gapcursor/style/gapcursor';
+@import '~prosemirror-tables/style/tables';
+
+@import 'components/Editor/styles/latex.scss';
```

Comment:
Should this be scoped to `.editor` as well?

Reply:
I don't think so? It isn't in the editor base.

Reply:
Thinking that was probably an oversight, since this style comes from the former editor package.